File: leetcode/1easy/이경호/2928.cpp
class Solution {
  public:
      int distributeCandies(int n, int limit) {
          int ans = 0;
          for(int i = 0; i <= n; i++) {
              if(i > limit) {
                  continue;
              }
              for(int j = 0; j <= n - i; j++) {
                  if(j > limit || n - i - j > limit) {
                      continue;
                  }
                  ans++;
              }
          }
          return ans;
      }
  };

File: leetcode/1easy/김민진/2928.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def distributeCandies(self, n: int, limit: int) -> int:
        count = 0
        for d1 in range(min(n, limit) + 1):
            # count += (min(n-d1, limit) + 1)
            for d2 in range(min(n-d1, limit) + 1):
                if n-d1-d2 <= limit:
                    count += 1
        return count
        